sig-scalability/README.md

```diff
@@ -49,6 +49,10 @@ subprojects, and resolve cross-subproject technical issues and decisions.
 ## Subprojects
 
 The following [subprojects][subproject-definition] are owned by sig-scalability:
+### inference-perf
+[Described below](#inference-perf)
```

Comment:
Can you please add a section for that, just leave a TODO in this section to fill it in

Reply:
Done